manim/scene/scene_file_writer.py

```diff
@@ -79,6 +81,14 @@ def __init__(self, renderer, scene_name, **kwargs):
         self.next_section(
             name="autocreated", type=DefaultSectionType.NORMAL, skip_animations=False
         )
+        # fail fast if ffmpeg is not found
+        if not ensure_executable(Path(config.ffmpeg_path)):
+            logger.critical(
+                """FFmpeg not found.
+Manim requires an ffmpeg installation to render videos.
+Please download ffmpeg """
+            )
+            sys.exit(1)
```

Comment:
What was the result of your Discord poll? Personally, I think this should raise a proper exception.

Reply:
like `raise SystemExit("FFmpeg Not Found")` ?
sys.exit already raises that, but sure I can make it more explicit 👍

Reply:
The issue of ffmpeg missing should be in the message of the error. Judging from how little the error logs regarding LaTeX build errors are read, something more explicit than a generic `sys.exit(1)` would be good.

Reply:
I also think raising a proper exception is better than calling `sys.exit(1)`. Say, a `RuntimeError` should do.

Reply:
+1 for `RuntimeError`, and then we can move the error message from the logger to the error.

Reply:
But that would report a `Traceback` and I do not think a Traceback in this case would be helpful. What would a third party program do if their system doesn't have ffmpeg in the first place, can they handle such an exception?
I can see how it might be useful when you would want to bootstrap manim deployment using python, but I do not think people would do that too often, the added noise of an unhelpful traceback in this case doesn't seem worth it to me.

Reply:
Can you do me the favor and post the terminal output of both variants, detailed error message in the logger + `sys.exit` vs. detailed error message in `RuntimeError`? I just want the variant where the error is communicated as clearly as possible.

Reply:
alright, 
Here's `logger + sys.exit`

<img width="1440" alt="Screenshot 2022-07-09 at 1 29 05 PM" src="https://user-images.githubusercontent.com/90276965/178097217-93ed5530-f322-4e63-9033-1ea5ad6a683f.png">

Here's `RuntimeError`

<img width="867" alt="Screenshot 2022-07-09 at 1 30 31 PM" src="https://user-images.githubusercontent.com/90276965/178097255-a9737775-09f6-4c92-ab52-2cd4f707da4e.png">

Reply:
I have added an exception, instead of `logger + sys.exit()`